src/PrestaShopBundle/Resources/config/services/core/form/form_builder.yml

```diff
@@ -0,0 +1,16 @@
+services:
```

Comment:
Note: I think we should do the same kind of organisation for "settings form", but not in this PR of course

Reply:
yeah of course it's optional, this can be done . in a future PR

Reply:
Argh 😅 it's a never-ending debate, gather things by type or feature
Same issue applies to :
- do you put all FormTypes in same folder or do you put ProductFormType with ProductDataHandler and ProductDataProvider
- do you put all Exceptions in same folder or do you put SqlException with SqlManager and SqlGetRequest

We'll see with @eternoendless 😉 in a future PR indeed

Reply:
I think it's logical to separate classes by type (all DataProvider are in the same namespace, all FormType are in same namespace, and so on)
but for service declarations as it's configuration maybe it would be easier to regroupe them by feature